### ISSUE TITLE
Crash when I open message (that I send to wanderer)

### ISSUE THREAD

**Issue by Majber** [DiplomacyTeam/Bannerlord.Diplomacy#171]
Every time I send a message to wanderer and I hear from him and click to talk to him I've got crash.
Im using latest mod and game versions, graphic drivers etc. That's so far only repeatable crash for me.

here is my crash report:
https://mega.nz/file/XwVQmLLT#UZFHOUrUZ5VfwCQ70kNRux50fwyy9eFNbsG_7eeSBzw
here is my mod list:
https://imgur.com/a/5sL7wa5
logs from ProgramData\Mount and Blade II Bannerlord\logs:
[rgl_log_2748.txt](https://github.com/DiplomacyTeam/Bannerlord.Diplomacy/files/10303916/rgl_log_2748.txt)
[rgl_log_errors_2748.txt](https://github.com/DiplomacyTeam/Bannerlord.Diplomacy/files/10303917/rgl_log_errors_2748.txt)
Link to save game:
https://mega.nz/file/H1UlGaDT#X4_6R-cIlYcD-mbPmSAMrVmZul59ufjARSF6YC_QzFA

If you want anythink more just ask.

**Comment by artifixer** [comment]
I don't see Diplomacy in the call stack. Are you sure it's tied to the messengers?

**Comment by Majber** [comment]
Yes, crash happen after I click on notification that messenger is back (probably some talking option should show after that click, like you are talking to noc?)
Maybe some other mod is interrupting this? I don't know.

**Comment by Majber** [comment]
If its not your mod fault can you help me be ouse I don't even know on what mod bug section I should report it then...

**Comment by artifixer** [comment]
The report has RBM patches in call stack. It might be incompatible with messaging someone in settlements. Or it might be a Diplomacy problem. Or some combination of both. Have not looked into it yet, and probably won't look this week.

**Comment by Majber** [comment]
ok, but if you will have time it would be great :)

**Comment by VictorValar** [comment]
I'm facing the exact same issue!
